metaBuild.sh

```diff
@@ -8,7 +8,7 @@ if [ "${ARG1}" = "clean" ]; then
   exit 0
 fi
 
-if [ "${ARG2}" = "release" ]; then
+if [ "${ARG1}" = "release" ]; then
```

Comment:
I'd suggest reverting metaBuild.sh to the version in master so that it matches upstream instead, ie: https://github.com/bantamtools/geode/blob/master/metaBuild.sh